models/user/user.go

```diff
@@ -506,8 +527,17 @@ func IsUserExist(uid int64, name string) (bool, error) {
 }
 
 // GetUserSalt returns a random user salt token.
+//
+// Note: As of the beginning of 2022, it is recommended to use at least
+// 64 bits of salt, but NIST is already recommending to use to 128 bits.
+// (16 bytes = 16 * 8 = 128 bits)
 func GetUserSalt() (string, error) {
-	return util.RandomString(10)
+	rBytes, err := util.RandomBytes(16)
```

Comment:
We should move this to constant (16) and then use it everywhere we do compare also to 32 (2*constant) to be more future proof if we need to change it in the future